core/src/main/java/com/linecorp/armeria/common/ClientCookieDecoder.java

```diff
@@ -230,9 +230,14 @@ private static void parse7(CookieBuilder builder, String header,
     }
 
     private static void parse8(CookieBuilder builder, String header,
-                               int nameStart) {
+                               int nameStart, int valueStart, int valueEnd) {
         if (header.regionMatches(true, nameStart, CookieHeaderNames.HTTPONLY, 0, 8)) {
             builder.httpOnly(true);
+        } else if (header.regionMatches(true, nameStart, CookieHeaderNames.SAMESITE, 0, 8)) {
+            final String sameSite = computeValue(header, valueStart, valueEnd);
```

Comment:
What do you think of adding validation to the value of `SameSite`? I think the only available values are `Lax`, `Strict`, `None` (ignoring case)
https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis-05#section-5.3.7
https://github.com/netty/netty/blob/cccce01497db4ac33b412dfb60630fc5ab443ce1/codec-http/src/main/java/io/netty/handler/codec/http/cookie/ClientCookieDecoder.java#L251

I think it's not a bad idea to fork Netty's implementation 😄 
https://github.com/netty/netty/blob/cccce01497db4ac33b412dfb60630fc5ab443ce1/codec-http/src/main/java/io/netty/handler/codec/http/cookie/CookieHeaderNames.java#L37

Reply:
Thanks for your suggestion. It sounds good. I will add a validation. 😄